### ISSUE TITLE
Function log returns nothing

### ISSUE THREAD

**Issue by bif002** [fission/fission#1550]
Environment:
**Fission : 1.8.0
Env: NodeJs
Mikikube: 1.7.3 (--vm-driver=none)
Kubectl: 1.17.3
Docker: 18.0.9**

Sample Node function:

```nodejs
module.exports = async function(context) {
    console.log('Function log ...');
    return {
        status: 200,
        body: "hello, world!\n"
    };
}
```
`fission function create --name hello --code hello.js --env nodejs

`fission function test --name hello`

**Expected result**
`fission fn logs --name hello`  should return `Function log ...` along with logger format

**Actual result**
`fission fn logs --name hello` nothing returned

**Additional info**
`kubectl -n fission-function get pod -l functionName=hello`
`kubectl -n fission-function logs -f poolmgr-nodejs-default-26873-5d4575f467-4w9sk nodejs`

actually returns the logged data but
`kubectl logs --namespace fission pod/logger-8pwpj  logger`
shows the following error
```
{"level":"error","ts":1583341110.3361034,"caller":"logger/logger.go:100","msg":"error creating symlink","container":"nodejs","pod":"poolmgr-nodejs-default-26873-5d4575f467-phs5p","namespace":"fission-function","error":"symlink /var/log/containers/poolmgr-nodejs-default-26873-5d4575f467-phs5p_fission-function_nodejs-efb1fbc6906a84426a2073013617fc946b9ec66e7bab8d8e5c8e37365a08d9ff.log /var/log/fission/poolmgr-nodejs-default-26873-5d4575f467-phs5p_fission-function_nodejs-efb1fbc6906a84426a2073013617fc946b9ec66e7bab8d8e5c8e37365a08d9ff.log: file exists","stacktrace":"github.com/fission/fission/pkg/logger.createLogSymlinks\n\t/go/src/pkg/logger/logger.go:100\ngithub.com/fission/fission/pkg/logger.makePodLoggerController.func2\n\t/go/src/pkg/logger/logger.go:68\nk8s.io/client-go/tools/cache.ResourceEventHandlerFuncs.OnUpdate\n\t/go/pkg/mod/k8s.io/client-go@v12.0.0+incompatible/tools/cache/controller.go:202\nk8s.io/client-go/tools/cache.newInformer.func1\n\t/go/pkg/mod/k8s.io/client-go@v12.0.0+incompatible/tools/cache/controller.go:362\nk8s.io/client-go/tools/cache.(*DeltaFIFO).Pop\n\t/go/pkg/mod/k8s.io/client-go@v12.0.0+incompatible/tools/cache/delta_fifo.go:436\nk8s.io/client-go/tools/cache.(*controller).processLoop\n\t/go/pkg/mod/k8s.io/client-go@v12.0.0+incompatible/tools/cache/controller.go:150\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190612205821-1799e75a0719/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190612205821-1799e75a0719/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/pkg/mod/k8s.io/apimachinery@v0.0.0-20190612205821-1799e75a0719/pkg/util/wait/wait.go:88\nk8s.io/client-go/tools/cache.(*controller).Run\n\t/go/pkg/mod/k8s.io/client-go@v12.0.0+incompatible/tools/cache/controller.go:124\ngithub.com/fission/fission/pkg/logger.Start\n\t/go/src/pkg/logger/logger.go:186\nmain.runLogger\n\tsrc/cmd/fission-bundle/main.go:99\nmain.main\n\tsrc/cmd/fission-bundle/main.go:293\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:200"}
```

but `kubectl logs --namespace fission pod/logger-8pwpj fluentbit`  looks ok

```
Fluent Bit v1.0.4
Copyright (C) Treasure Data
[2020/03/04 13:24:20] [ info] [storage] initializing...
[2020/03/04 13:24:20] [ info] [storage] in-memory
[2020/03/04 13:24:20] [ info] [storage] normal synchronization mode, checksum disabled
[2020/03/04 13:24:20] [ info] [engine] started (pid=1)
[2020/03/04 13:24:20] [ info] [filter_kube] https=1 host=kubernetes.default.svc.cluster.local port=443
[2020/03/04 13:24:20] [ info] [filter_kube] local POD info OK
[2020/03/04 13:24:20] [ info] [filter_kube] testing connectivity with API server...
[2020/03/04 13:24:20] [ info] [filter_kube] API server connectivity OK
```

**Comment by vishal-biyani** [comment]
@bif002 Does your Fission installation also has InfluxDB which comes with fission-all chart?

**Comment by bif002** [comment]
@vishal-biyani  - yes, i have installed fission-all chart from helm.

**Comment by vishal-biyani** [comment]
This will be fixed by https://github.com/fission/fission/pull/1671 also reported in https://github.com/fission/fission/issues/1634

**Comment by vishal-biyani** [comment]
Closed by #1671
